cli/cli/helpers/engine_manager/engine_existence_guarantor.go

```diff
@@ -194,7 +206,13 @@ func (guarantor *engineExistenceGuarantor) VisitContainerRunningButServerNotResp
 }
 
 func (guarantor *engineExistenceGuarantor) VisitRunning() error {
-
+	//TODO(centralized-logs-infra-deprecation) remove this code in the future when people don't have centralized logs infra running
```

Comment:
What's this TODO for? I am not sure I fully understand it

Reply:
not sure tbh